src/main/native/include/sysid/view/Analyzer.h

```diff
@@ -86,5 +86,9 @@ class Analyzer : public glass::View {
 
   // Plot
   AnalyzerPlot m_plot{m_logger};
+
+  float m_graphScroll = 0.0;
+  bool m_graphLoaded = false;
+  bool m_dataLoaded = false;
```

Comment:
Since this is modified in the thread.
```suggestion
  std::atomic<bool> m_dataLoaded{false};
```